Makefile

```diff
@@ -24,7 +24,7 @@ ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=BluzelleService \
 	-X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT) \
 	-X "github.com/cosmos/cosmos-sdk/version.BuildTags=$(build_tags_comma_sep),cosmos-sdk $(COSMOS_SDK)"
 
-BUILD_FLAGS := -tags "$(build_tags)" -ldflags '$(ldflags)'
+BUILD_FLAGS := -tags "$(build_tags)" -ldflags '$(ldflags)' -tags faucet
```

Comment:
Does this update the version info as well?

Reply:
No. Should it?

{"name":"BluzelleService","server_name":"blzd","client_name":"blzcli","version":"0.0.0-40-g62b476f","commit":"62b476f774c9788e8777f5346007a5e05095eebc","build_tags":"ledger,cosmos-sdk v0.38.1","go":"go version go1.13.6 darwin/amd64"}

Reply:
Now it does.